src/Raven.Server/Web/Studio/SqlMigrationHandler.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Raven.Server.Documents;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.SqlMigration;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Studio
+{
+    public class SqlMigrationHandler : DatabaseRequestHandler
+    {
+        [RavenAction("/databases/*/admin/sql-schema", "POST", AuthorizationStatus.DatabaseAdmin)]
```

Comment:
Can I suggest changing the endpoint to:
/databases/*/admin/sql-migration/schema ?

Also this looks more like a GET endpoint?